tests/mocha/comment_test.js

```diff
@@ -133,4 +146,71 @@ suite('Comments', function() {
       assertBubbleSize(this.comment, 100, 100);
     });
   });
+  suite('Restore Comment', function() {
+    setup(function() {
+      this.blockCount = this.workspace.getAllBlocks().length;
+      this.tempBlock = Blockly.Xml.domToBlock(Blockly.Xml.textToDom(
+        '<block type="empty_block"/>'
+      ), this.workspace);
+      this.tempBlock.setCommentText('test temp text');
+    });
+    function assertComment(workspace, text) {
+      const count = workspace.getAllBlocks().length;
+      // show comment
```

Comment:
Start with a capital and end with a period =) (Here and below)
```suggestion
      // Show comment.
```

Reply:
Fixed as suggested.

Reply:
Thank you for all the fixes!